ZenPacks/zenoss/CalculatedPerformance/__init__.py

```diff
@@ -45,7 +45,10 @@ class ZenPack(ZenPackBase):
     """
     ZenPack loader that handles custom installation and removal tasks.
     """
-    packZProperties = [('zAggregatorCollectionInterval', 300, 'int'), ('zDatasourceDebugLogging', False, 'boolean')]
+    packZProperties = [
+        ('zAggregatorCollectionInterval', 300, 'int'), 
+        ('zDatasourceDebugLogging', False, 'boolean'),
+        ('zUseBasisIntervals', True, 'boolean'),]
```

Comment:
I recommend not having a zUseBasisIntervals property. This should only be a per-datasource configuration. Having it in both places is confusing. Especially since the user is unable to choose between the three options of enabled/disabled/inherited when configuring datasources.